cdp_scrapers/legistar_utils.py

```diff
@@ -100,8 +100,8 @@
 ###############################################################################
 
 
-known_persons: Dict[int, Dict[str, Any]] = {}
-known_bodies: Dict[int, Dict[str, Any]] = {}
```

Comment:
why was known_bodies changed? I only see static persons not static bodies.

Reply:
This is actually renaming old code to avoid confusion with the new `known_persons`; old `known_persons` -> `known_legistar_persons`, old `known_bodies` -> `known_legistar_bodies`.

I noticed while working on https://github.com/CouncilDataProject/cdp-scrapers/pull/26 that a single `get_events()` call (and thus `get_legistar_events_for_timespan()`) would make 10s-100s of duplicate web queries for the exact same person / body. So I made these 2 variables as cache, for use during a single `get_events()` call. My thinking was that the probability of person / body info changing in `get_events()` is _highly_ minuscule